Shared/Enums.cs

```diff
@@ -87,10 +87,16 @@ public enum AwakeType : byte
 [Flags]
 public enum LevelEffects : byte
 {
-    None = 0,
-    Mist = 0x0001,
-    RedDragon = 0x0002,
-    BlueDragon = 0x0004
+     None = 0,
+    Mist = 1,
+    RedDragon = 2,
+    BlueDragon = 4,
+    Rebirth1 =8,
+    Rebirth2 = 16,
+    Rebirth3 = 32,
+    NewBlue = 64,
+    YellowDragon = 128,
+    Phoenix= 144,
```

Comment:
144 is wrong. The next number should be 256 - which means you'll need to change this LevelEffects to a ushort, and also change the packets to accept ushort too.